src/docs/admin/references/NodeSourceInfrastructureReference.adoc

```diff
@@ -769,7 +769,12 @@ You can opt to place this file in `$PROACTIVE_HOME/config/authentication/azure.c
 
 - *sshPublicKey:* Optional parameter. In case of Linux, use it to provide a public RSA key to connect to your VMs via SSH. In case of Windows, use it to define a password to connect to your VMs via Remote Desktop. In both cases, the default user is _activeeon_. If the field is left blank a password will be automatically generated and written in the Scheduler log.
 - *targetNetwork:* Optional parameter. Use it to define a specific Azure Virtual Network subnet for your Scale Set. If defined, the format should be _{resourceGroupName}:{network}:{subnet}_, e.g. _myGroup:myVNet:default_. If left unset, a new Virtual Network and subnet will be created inside the Resource Group.
-- *region:* Define the Azure region where the Resource Group and the Scale Set will be created. For a list of the regions supported by your subscription, you can use the https://docs.microsoft.com/en-us/cli/azure/account?view=azure-cli-latest#az-account-list-locations[CLI command] `az account list-locations -o table`, the right name to use is the one in the column _name_. The region name should be in lowercase without spaces, e.g. for the region _West US 2_ use _westus2_. Default value is _westeurope_.
+- *azureRegion:* Define the Azure region where the Resource Group and the Scale Set will be created. For a list of the regions supported by your subscription, you can use the https://docs.microsoft.com/en-us/cli/azure/account?view=azure-cli-latest#az-account-list-locations[CLI command] `az account list-locations -o table`, the right name to use is the one in the column _name_. The region name should be in lowercase without spaces, e.g. for the region _West US 2_ use _westus2_. Default value is _westeurope_.
```

Comment:
azureRegion is Optional parameter or not ? 
Define --> Defines
For a list --> For the list
table`, the right --> can you please make this sentence shorter?  I think that you can put `the right name to use is the one in the column _name_` in a new sentence.

Reply:
I only changed the *region:* to*azureRegion:* 
So I didn't write these sentences))
But we can still improve the doc according to your suggestions

Reply:
azureRegion is not Optional parameter